tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -115,7 +123,7 @@ def test_evaluation_list_limit(self):
         self.assertEqual(len(evaluations), 100)
 
     def test_list_evaluations_empty(self):
-        evaluations = openml.evaluations.list_evaluations("unexisting_measure")
+        evaluations = openml.evaluations.list_evaluations("unexisting_measure", size=None)
```

Comment:
```suggestion
        evaluations = openml.evaluations.list_evaluations("unexisting_measure")
```

Reply:
I had to add back `size=100` to this test because the assert statement right after this line verifies if the length of the results returned is 100.
Apart from this one discrepancy, I've made all the other changes as you'd specified & the tests ran succesfully on my local machine.